kaspresso/src/main/kotlin/com/kaspersky/kaspresso/testcases/core/testcontext/BaseTestContext.kt

```diff
@@ -26,4 +26,7 @@ open class BaseTestContext internal constructor(
     ContinuouslyProvider by ContinuouslyProviderImpl(kaspresso.params.continuouslyParams, kaspresso.libLogger),
     ComposeProvider by ComposeProviderImpl(kaspresso),
     WebComposeProvider by WebComposeProviderImpl(kaspresso),
-    TestAssistantsProvider by TestAssistantsProviderImpl(kaspresso)
\ No newline at end of file
+    TestAssistantsProvider by TestAssistantsProviderImpl(kaspresso) {
+
+    val params = kaspresso.params
```

Comment:
This should be in `TestAssistantsProviderImpl`, not here.

Reply:
I think it's a more appropriate way.

Reply:
Fixed